input/data/2009/2009A%26A...499..723A/tev-000132.yaml

```diff
@@ -1,4 +1,26 @@
 source_id: 132
 reference_id: 2009A&A...499..723A
 
-# TODO: add data
+data:
+  # TODO: No livetime found in the paper
+  livetime:
+  significance: 12.1
+  excess: 689
+
+pos:
+  ra: {val: 40d23m9.2s, err: 2.4m, err_sys: 20s}
+  dec: {val: 0d47m10.1s, err: 2.4m, err_sys: 20s}
```

Comment:
In this case the position is given in the paper as l / b, i.e. in Galactic coordinates. (not in RA / DEC).
As mentioned at https://github.com/gammapy/gamma-cat/blob/master/input/schemas/dataset_source_info.schema.yaml#L45 for those cases we should use the key names `glon:` and `glat:`